Dockerfile.mqttwarn-slack

```diff
@@ -0,0 +1,19 @@
+# Docker build file for mqttwarn, with Slack SDK.
+# Based on https://hub.docker.com/r/jpmens/mqttwarn.
+#
+# Invoke like:
+#
+#   docker build --tag=mqttwarn-slack --file=Dockerfile.mqttwarn-slack .
+#
+
+# Derive from upstream image.
+FROM jpmens/mqttwarn:0.23.0
+
+# Make package installation run as "root" user
+USER root
+
+# Install Slack SDK.
+RUN pip install mqttwarn[slack]
```

Comment:
I am not aware of this syntax - can you explain how this works, and how you would write this to also install a second module - say `gpxpy`?

Reply:
I thnk it's a custom dockerfile, I think I saw somewhere that is gpxpy is another mqttwarn library it'd be: RUN pip install mqttwarn[slack,gpxpy]

So would need to be my own build....I might see if I can docker run jpmens/mqttwarn pip install mqttwarn[slack] instead as a one off, so I don't need my own docker image, as to be that seems to default the purpose of using the generic.

Reply:
yeah I meant the `pip install mqttwarn[<module>]` syntax - never seen that before.

Reply:
https://stackoverflow.com/questions/46775346/what-do-square-brackets-mean-in-pip-install

Reply:
Right - so the extra tag is referring to the modules defined in `setup.py` - so it wouldn't work by just adding `gpxpy` since that is included in `setup.py`.

Reply:
Hi Ben and @psyciknz,

thanks for signaling interest on this patch.

> I am not aware of this syntax.

Some Python packages have optional [extras](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#installing-extras), and so does `mqttwarn` in its [setup.py at the "extras" section](https://github.com/jpmens/mqttwarn/blob/0.23.0/setup.py#L20-L117) for all of the dependencies of the built-in service plugins.

> How you would write this to also install a second module?

You would invoke it like
```shell
pip install mqttwarn[slack,twitter]
```

With kind regards,
Andreas.

Reply:
Ok thanks @amotl - very nice.

Reply:
BTW - just tested this for my use-case and it works very nicely - very handy!